kessler/kessler_update.meta

```diff
@@ -0,0 +1,194 @@
+  intent = out
+[ errmsg ]
+  standard_name = ccpp_error_message
+  long_name = Error message for error handling in CCPP
+  units = 1
+  type = character | kind = len=512
+  dimensions = ()
+  intent = out
+[ errflg ]
+  standard_name = ccpp_error_flag
+  long_name = Error flag for error handling in CCPP
+  units = flag
+  type = integer
+  dimensions = ()
+  intent = out
+#########################################################
+[ccpp-arg-table]
+  name = kessler_update_run
+  type = scheme
+[ nz ]
+  standard_name = vertical_layer_dimension
+  long_name = number of vertical levels
+  units = count
+  dimensions = ()
+  type = integer
+  intent = in
+[ ncol ]
+  standard_name = horizontal_loop_extent
+  long_name = number of columns
+  units = count
+  dimensions = ()
+  type = integer
+  intent = in
+[ temp ]
+  standard_name = temperature
+  state_variable = true
+  type = real
+  kind = kind_phys
+  units = K
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
+  intent = in
+[ theta ]
+  standard_name = potential_temperature
+  long_name = potential temperature
+  units = K
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
+  type = real
+  kind = kind_phys
+  intent = inout
+[ exner ]
+  standard_name = inverse_exner_function_wrt_surface_pressure
+  long_name = inverse exner function w.r.t. surface pressure, (ps/p)^(R/cp)
+  state_variable = true
+  units = count
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
+  intent = in
+[ dt ]
+  standard_name = time_step_for_physics
+  long_name = time step
+  units = s
+  dimensions = ()
+  type = real
+  kind = kind_phys
+  intent = in
+[ temp_prev ]
+  standard_name = temperature_from_previous_timestep
+  units = K
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
+  type = real
+  kind = kind_phys
+  intent = inout
+[ ttend_t ]
+  standard_name = total_tendency_of_temperature
+  type = real
+  kind = kind_phys
+  units = K s-1
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
+  intent = inout
+[ errmsg ]
+  standard_name = ccpp_error_message
+  long_name = Error message for error handling in CCPP
+  units = 1
+  type = character | kind = len=512
+  dimensions = ()
+  intent = out
+[ errflg ]
+  standard_name = ccpp_error_flag
+  long_name = Error flag for error handling in CCPP
+  units = flag
+  type = integer
+  dimensions = ()
+  intent = out
+#########################################################
+[ccpp-arg-table]
+  name = kessler_update_timestep_final
+  type = scheme
+[ nz ]
+  standard_name = vertical_layer_dimension
+  long_name = number of vertical levels
+  units = count
+  dimensions = ()
+  type = integer
+  intent = in
+[ temp ]
+  standard_name = temperature
+  state_variable = true
+  type = real
+  kind = kind_phys
+  units = K
+  dimensions = (horizontal_dimension, vertical_layer_dimension)
+  intent = in
+[ zm ]
+  standard_name = geopotential
```

Comment:
I suspect this is geopotential height, not geopotential (as the units are in meters)